forge-gui/res/cardsfolder/upcoming/rakish_revelers.txt

```diff
@@ -0,0 +1,14 @@
+Name:Rakish Revelers
+ManaCost:2 R G W
+Types:Creature Elf Druid Rogue
+PT:5/3
+T:Mode$ ChangesZone | Origin$ Any | Destination$ Battlefield | ValidCard$ Card.Self | Execute$ TrigToken | TriggerDescription$ When CARDNAME enters the battlefield, create a 1/1 green and white Citizen creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ gw_1_1_citizen
+A:AB$ Effect | Cost$ 2 ExileFromHand<1/CARDNAME> | ActivationZone$ Hand | ValidTgts$ Land | TgtPrompt$ Select target land | RememberObjects$ Targeted,Self | StaticAbilities$ Land,MayPlay | Triggers$ Cast | ImprintCards$ Self | Duration$ Permanent | ForgetOnMoved$ Exile | SpellDescription$ Target land gains "{T}: Add {R}, {G}, or {W}" until CARDNAME is cast from exile. You may cast CARDNAME for as long as it remains exiled.
+SVar:Land:Mode$ Continuous | EffectZone$ Command | AffectedZone$ Battlefield | Affected$ Card.IsRemembered+IsNotImprinted | AddAbility$ Mana | Description$ Target land gains "{T}: Add {R}, {G}, or {W}" until Rakish Revelers is cast from exile. You may cast EFFECTSOURCE for as long as it remains exiled.
```

Comment:
Ideally this should be Animate I think if we had a Duration for that

scenarios where it might matter should be rare though, so not sure that's worth the effort

Reply:
💯 my thought process... I had an Animate line going and then opted for this instead 😆